app/templates/people/index.hbs

```diff
@@ -136,7 +136,11 @@
             </td>
             <td>
               {{#each person.organizations as |organization index|}}
-                {{#if organization.id}}
+                {{#if (and
+                  organization.id
+                  (not-eq organization.classification "Autonoom gemeentebedrijf")
```

Comment:
this part *

Reply:
Initially didn't do it because of the imbrication of `{{#each person.organizations as |organization index|}}` :grimacing: but you're right that id would make more sense, I'll have to create a component for it then I think